addons/webinterface.default/lib/video-js/video.js

```diff
@@ -2264,7 +2264,7 @@ var Component = function () {
   /**
    * Clears a timeout that gets created via `window.setTimeout` or
    * {@link Component#setTimeout}. If you set a timeout via {@link Component#setTimeout}
-   * use this function instead of `window.clearTimout`. If you don't your dispose
+   * use this function instead of `window.clearTimeout`. If you don't your dispose
```

Comment:
These changes should have gone also upstream to chorus2

Reply:
what is the repo address for upstream ?

Reply:
https://github.com/xbmc/chorus2

Reply:
yes, missed it
https://github.com/xbmc/chorus2/blob/43750e50663ac75f671779ed998a279f31ff6023/dist/lib/video-js/video.js#L2267

Reply:
Opened PR in chorus: https://github.com/xbmc/chorus2/pull/451